slave-maven/Dockerfile.rhel7

```diff
@@ -2,7 +2,7 @@ FROM openshift/jenkins-slave-base-rhel7
 
 MAINTAINER Ben Parees <bparees@redhat.com>
 
-ENV MAVEN_VERSION=3.3.9
+ENV MAVEN_VERSION=3.0.5
```

Comment:
nah, we were just never actually at 3.3.9. We are installing 3.3.9 in centos, but the rhel package is 3.0.5, so i was just fixing the misinformation.